### ISSUE TITLE
iOS 9.9.10.1 (14B55c) 

### ISSUE THREAD

**Issue by tpurtell** [kpwn/yalu102#447]
let me just leave this here
`
    else if (strcmp(u.version, "Darwin Kernel Version 16.1.0: Fri Sep 16 03:53:23 PDT 2016; root:xnu-3789.20.46~54/RELEASE_ARM64_T7000") == 0) {
        allproc_offset = 0x5B0168;
        rootvnode_offset = 0x5B60B8;
    }
`

**Comment by tangalbert919** [comment]
This looks very useful @tpurtell, and you actually made this issue code-related by adding code.
I have 2 questions for this:
1. Who would be on iOS 10.1 beta 1 right now?
2. Why didn't you just add this to the "offsets.c" file in this project?

**Comment by tpurtell** [comment]
Unfortunately I heard about Yalu after the signing window had closed.  I happened to have a test phone on  the beta so I did the analysis to find the offsets.   I suppose there might be a few other lucky souls out there who happen to have an old OS install they can work with, but who knows.

The code is intended to be added there, but since the window has closed, I didn't send it as a PR as the repository seems to be silent now.  The real directions to do this for other firmware's appear to be here: https://github.com/kpwn/yalu102/issues/414
